.kokoro/dependencies.sh

```diff
@@ -36,3 +36,16 @@ retry_with_backoff 3 10 \
     -Dclirr.skip=true
 
 mvn -B dependency:analyze -DfailOnWarning=true
+
+echo "****************** DEPENDENCY LIST COMPLETENESS CHECK *******************"
+source ${scriptDir}/completeness-check.sh
+
+## Locally install artifacts
+mvn verify -DskipTests
```

Comment:
We shouldn't need this line, it's being installed on line 33

Reply:
Done